pygpi/entry.py

```diff
@@ -0,0 +1,18 @@
+import os
+import importlib
+from functools import reduce
+
+
+def load_entry():
```

Comment:
I'm not sure how useful this actually is:
```suggestion
from types import ModuleType
from typing import Tuple, Callable


def load_entry() -> Tuple[ModuleType, Callable]:
```